### PR TITLE
Change deploy commands to status slash commands

### PR DESCRIPTION
**Pull request by alanna-zhou**

### CODE REVIEW

File: deploybot.py [hunk 1]
```diff
@@ -14,49 +14,46 @@ def is_request_valid(request):
...
+
+@app.route('/uptime/', methods=['POST'])
+def uptime():
+  return status('uptime')
+
+@app.route('/pem/', methods=['POST'])
+def pem():
   if not is_request_valid(request):
-    return response('Sorry! Deploying is not available for you.')
+    return response('Sorry! /pem is not avilable for you.')
```
Comment (meganle) [inline deploybot.py]: available

File: deploybot.py [hunk 2]
```diff
@@ -14,49 +14,46 @@ def is_request_valid(request):
...
-  return response(
-      'Sorry! Could not deploy {}. Try one of these: {}'.format(
-           text, ', '.join([a for a in data['SLACK']['APPS']])
-      )
-  )
+    if request.form.get('user_id', '') in data['SLACK']['USER_IDS'][text]:
+      return response('Getting /pem for {}'.format(text))
+    return response('Sorry! /pem {} is not available for you'.format(text))
+  return response('Sorry! Could not get /pem for {}. Try one of these: {}'.format(
+      text, ', '.join([a for a in data['SLACK']['APPS']])
```
Comment (meganle) [inline deploybot.py]: could you just do `data['SLACK']['APPS']` instead of `[a for a in data['SLACK']['APPS']]`

File: deploybot.py [hunk 3]
```diff
@@ -14,49 +14,46 @@ def is_request_valid(request):
...
+    return response('Sorry! Getting {} is not available for you.'.format(status_type))
+  text = request.form.get('text', '').lower()
+  if not text:
+    return response('Please add an app to get {} for. Ex. /{} eatery'.format(
+        status_type, status_type
+    ))
+  if text in data['SLACK']['APPS']:
+    return response('Getting {} for {}...'.format(status_type, text))
+  return response('Sorry! Could not get {} for {}. Try one of these: {}'.format(
+      status_type, text, ', '.join([a for a in data['SLACK']['APPS']])
```
Comment (meganle) [inline deploybot.py]: same question as above

**Comment by cswenberg** [comment]
lgtm once you make Megan's changes

File: deploybot.py (hunk 2 above)
Comment (alanna-zhou) [reply deploybot.py]: it wouldn't look formatted and nice -- i tried with just `data['SLACK']['APPS']` but it looks ugly with the 'u''s and brackets

Comment (kq-li) [reply deploybot.py]: what do you mean -- what sort of data is in `data['SLACK']['APPS']`?

Comment (alanna-zhou) [reply deploybot.py]: it's just the list of apps like "eatery, transit, pollo, ..."

Comment (kq-li) [reply deploybot.py]: what does it respond with when you do `', '.join(data['SLACK']['APPS'])`?

Comment (alanna-zhou) [reply deploybot.py]: `', '.join(data['SLACK']['APPS'])` gives "eatery, transit, pollo..." but just `data['SLACK']['APPS']` gives [u'eatery', u'transit', u'pollo', ...]

Comment (kq-li) [reply deploybot.py]: right yeah just do the first thing -- the list comprehension here is redundant (`[a for a in ...]`) and you can just use the list directly

**Pull request merged**